takwimu/settings.py

```diff
@@ -97,6 +97,15 @@
     'UA-44795600-8',  # HURUmap
 ]
 
+HURUMAP['primary_release_year'] = {
+    'country': 2017,
+}
+HURUMAP['latest_release_year'] = '2017'
+HURUMAP['primary_dataset_name'] = 'Census'
+HURUMAP['available_release_years'] = {
+    'country': [2017]
+}
+
```

Comment:
Somehow this comment got dropped.

I was thinking, here in `HURUMAP['available_release_years']`, we'll define all years for all countries. This means if census in Tanzania was in 2012, and 2009 in Nigeria, this will contain `[2012, 2009]`.

We'll then define `HURUMAP['tz']['available_release_years']` that will only contain available years for Tanzania i.e. `[2012]`. We'll then use `HURUMAP['tz']['available_release_years']` when in Tanzania to compute `primary_releases.other` to list releases in the releases selection dropdown.